manifests/freshclam.pp

```diff
@@ -31,7 +31,7 @@
   # NOTE: RedHat comes with /etc/cron.daily/freshclam instead of a service
   if $freshclam_service {
     service { 'freshclam':
-      ensure     => running,
+      ensure     => $clamd::freshclam_service_ensure,
       name       => $freshclam_service,
```

Comment:
Same here.  You are using the var scoped from the clamd class, which I am OK with, if all vars do this....
Otherwise continue the same format and use the local scope var.